resources/js/pack-it-forms.js

```diff
@@ -1,5 +1,6 @@
 /* Copyright 2014 Keith Amidon
    Copyright 2014 Peter Amidon
+   Copyright 2018 John Kristian
```

Comment:
Reviewing this points out that Peter and I haven't been very diligent about updating our copyright years. We'll have to fix that in a future commit.